manifests/plugin/remote_execution/ssh_user/params.pp

```diff
@@ -0,0 +1,31 @@
+# Remote Execution SSH user default parameters
+class foreman_proxy::plugin::remote_execution::ssh_user::params {
+
+  if defined('$::remote_execution_create_user') {
+    $remote_execution_create_user = str2bool($::remote_execution_create_user)
```

Comment:
I think it makes more sense to name the variable `manage_user` to have parity with the class parameter

Reply:
Fixed.  All variables in `params.pp` now match the class variable names.